src/debug/createdump/threadinfo.cpp

```diff
@@ -265,6 +290,20 @@ ThreadInfo::GetRegistersWithDataTarget(ICLRDataTarget* pDataTarget)
 
     assert(sizeof(context.FltSave.XmmRegisters) == sizeof(m_fpRegisters.xmm_space));
     memcpy(m_fpRegisters.xmm_space, context.FltSave.XmmRegisters, sizeof(m_fpRegisters.xmm_space));
+#elif defined(__aarch64__)
```

Comment:
FYI, this function (GetRegistersWithDataTarget) is only called from the coreclr SOS in the "createdump" command.  Did you try the coreclr SOS (not the new diagnostics repo one) "createdump" command. Not sure how important it is to keep this command working since the coreclr SOS is going away eventually.

Reply:
I didn't. I'll try this out. Thanks for pointing it out Mike.